AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -105,6 +105,26 @@
     'NumRaysPerProbe_864': 5,
     'NumRaysPerProbe_1008': 6,
 }
+
+# LUT Resolution options for the HDR Color Grading component.
+LUT_RESOLUTION = {
+    '16x16x16': 0,
+    '32x32x32': 1,
+    '64,64,64': 2,
```

Comment:
is there a reason this it '64,64,64' instead of '64x64x64' I'm assuming its just that way in the engine. seems like a bug.